db/migrate/20170601233834_delete_available_column_from_game.rb

```diff
@@ -0,0 +1,5 @@
+class DeleteAvailableColumnFromGame < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :games, :available, :boolean
```

Comment:
If a migration has been merged to master/deployed. Then yes - definitely make a new migration.

But if it's only in a branch. Then it can be rolled back.